.github/workflows/update-gradle-wrapper.yml

```diff
@@ -0,0 +1,25 @@
+name: Update Gradle Wrapper
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+  workflow_dispatch:
+
+jobs:
+  update-gradle-wrapper:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the code
+        uses: actions/checkout@v4
+
+      - name: Set up Java
+        uses: actions/setup-java@v4
+        with:
+          distribution: "temurin"
+          java-version: 21
+
+      - name: Grant execute permission for gradlew
+        run: chmod +x gradlew
```

Comment:
What is the reason for including this step? GitHub should remember the permissions.

Reply:
> What is the reason for including this step? GitHub should remember the permissions.

In some repositories, it fails if this step isn't exists. But removed as you asked.